pkg/api/types.go

```diff
@@ -503,12 +503,21 @@ type HostedMasterProfile struct {
 	Subnet string `json:"subnet"`
 }
 
+type AuthenticatorType string
+
+const (
+	OIDC    AuthenticatorType = "OIDC"
```

Comment:
To fix lint, you could just convert these to `oidc` and `webHook` because they don't seem to be consumed as exported consts currently

Reply:
@jackfrancis fixed. Pushed new commit.